tools/cosmovisor/README.md

```diff
@@ -60,22 +61,11 @@ go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v0.1.0
 
 Run `cosmovisor version` to check the cosmovisor version.
 
-You can also install from source by pulling the cosmos-sdk repository and switching to the correct version and building as follows:
+Alternatively, for building from source, simply run `make cosmovisor`. The binary will be located in `tools/cosmovisor`.
 
-```shell
-git clone git@github.com:cosmos/cosmos-sdk
-cd cosmos-sdk
-git checkout cosmovisor/vx.x.x
-make cosmovisor
-```
-
-This will build cosmovisor in `/cosmovisor` directory. Afterwards you may want to put it into your machine's PATH like as follows:
-
-```shell
-cp cosmovisor/cosmovisor ~/go/bin/cosmovisor
-```
-
-*Note: If you are using go `v1.15` or earlier, you will need to use `go get`, and you may want to run the command outside a project directory.*
+:::warning
+Building from source using `make cosmovisor` won't display the correct `cosmovisor` version.
```

Comment:
could we include the version in the binary the same way we do in the sdk?

Reply:
If we do that with ldflags, then it won't show in go install.